bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -305,14 +288,14 @@ public void startElement(@Nullable String uri, @Nullable String localName, @Null
                     if (curIgnore == null) {
                         curIgnore = new ArrayList<>();
                         curIgnore.add("DIDL-Lite");
-                        curIgnore.add("type");
-                        curIgnore.add("ordinal");
-                        curIgnore.add("description");
+                        curIgnore.add("r:type");
+                        curIgnore.add("r:ordinal");
+                        curIgnore.add("r:description");
```

Comment:
Are you sure that it is "r" ?

Reply:
Yes, I remember debugging and testing this. This was one side-effect of using the new API.